### PR TITLE
chore(release): trigger build image pipeline

### PR DESCRIPTION
**Pull request by TomasSlama**
### Description

Docker image needs to be built before we try to deploy storybook

### Review

- [ ] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/api-principles.md)
- [ ] Annotate all `props` in component with documentation
- [ ] Create `examples` for component
- [ ] Ensure that deployed demo has expected results and good examples
- [ ] Ensure that tests pass by running `yarn test`
- [ ] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [ ] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

**Comment by TomasSlama** [comment]
we need to wait for `TOPTAL_JENKINS_BUILD_TOKEN` to be added to [secrets](https://github.com/toptal/picasso/settings/secrets/actions)

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -103,6 +103,25 @@ jobs:
...
+          jenkins_token: ${{ env.TOPTAL_JENKINS_BUILD_TOKEN }}
+          proxy: ${{ env.HTTP_PROXY }}
+          job_name: ${{ env.JENKINS_JOB_NAME }}
+          job_params: |
+            {
+              BRANCH: "master",
+              VERSION: "${{ github.sha }}",
+              IMAGE_NAME: "${{ env.REPOSITORY_NAME }}"
+            }
+          job_timeout: '3600'
```
Comment (denieler) [inline .github/workflows/release.yml:123]: can we maybe build this image inside this job? ex. https://github.com/toptal/modularity-template/blob/4f11a86a5fe20963389b09c23db6ce01f0f3d596/.github/workflows/deploy-staging.yml#L139

Comment (dmaklygin1) [reply .github/workflows/release.yml:123]: wdyt about this solution? 
https://github.com/toptal/talent-activation-frontend/blob/master/.github/workflows/pr.yml#L106

**Review by dmaklygin1 — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (denieler) [reply .github/workflows/release.yml:123]: let's add maybe a ticket to Picasso backlog to improve this later?

Comment (TomasSlama) [reply .github/workflows/release.yml:123]: @denieler https://toptal-core.atlassian.net/browse/FX-2267

**Pull request merged**